reflex/components/component.py

```diff
@@ -544,6 +552,10 @@ def _render(self, props: dict[str, Any] | None = None) -> Tag:
         props.update(self._get_style())
         props.update(self.custom_attrs)
 
+        # remove excluded props from prop dict before adding to tag.
+        for prop_to_exclude in self._exclude_props():
+            del props[prop_to_exclude]
```

Comment:
This should be a .pop(prop_to_exclude, None)

In case the named prop didn't get added for whatever reason.